flink-runtime/src/main/java/org/apache/flink/runtime/security/token/DefaultDelegationTokenManager.java

```diff
@@ -58,6 +59,13 @@
 @Internal
 public class DefaultDelegationTokenManager implements DelegationTokenManager {
 
+    private static final String PROVIDER_RECEIVER_INCONSISTENCY_ERROR =
+            " There is an inconsistency between loaded delegation token providers and receivers. "
+                    + "One must implement a DelegationTokenProvider and a DelegationTokenReceiver "
+                    + "with the same service name and put them together to the classpath to make "
+                    + "the system consistent. The mentioned classes are loaded with Java's service "
+                    + "loader so the appropriate META-INF registration must not be forgotten.";
```

Comment:
nit: must not be forgotten -> also needs to be created.

Reply:
Fixed.